packages/crashlytics/lib/handlers.js

```diff
@@ -78,6 +81,21 @@ export const setGlobalErrorHandler = once(nativeModule => {
     if (nativeModule.isErrorGenerationOnJSCrashEnabled) {
       try {
         const stackFrames = await StackTrace.fromError(error, { offline: true });
+
+        // Flag the Crashlytics backend that we have a fatal error, they will transform it
+        await nativeModule.setAttribute(FATAL_FLAG, Math.floor(Date.now() / 1000)); // "current UNIX time"
+
+        // Notify analytics, if it exists
+        const analytics = firebase.app().analytics();
```

Comment:
I think this will throw if its not actually there? I could be wrong though, we used to check NativeModules export from RN instead to check existence of a module to get around the throw